From dd3bd9d63f22923de6bc66b9181d4da2c4f466a7 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 17 Dec 2021 15:02:57 +0000
Subject: [PATCH 01/14] Revert "init: Do not set safety net props for ENG
 builds"

This reverts commit 1fb7721be8ef961f9dc8407b95f085ac8f3aca3d.
---
 init/Android.bp           | 3 ---
 init/Android.mk           | 6 ++----
 init/property_service.cpp | 4 +---
 3 files changed, 3 insertions(+), 10 deletions(-)

diff --git a/init/Android.bp b/init/Android.bp
index 8ff968a1c..4865694f3 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -92,7 +92,6 @@ cc_defaults {
         "-DDUMP_ON_UMOUNT_FAILURE=0",
         "-DSHUTDOWN_ZERO_TIMEOUT=0",
         "-DINIT_FULL_SOURCES",
-        "-DSPOOF_SAFETYNET=1",
     ],
     product_variables: {
         debuggable: {
@@ -115,8 +114,6 @@ cc_defaults {
             cppflags: [
                 "-USHUTDOWN_ZERO_TIMEOUT",
                 "-DSHUTDOWN_ZERO_TIMEOUT=1",
-                "-USPOOF_SAFETYNET",
-                "-DSPOOF_SAFETYNET=0",
             ],
         },
         uml: {
diff --git a/init/Android.mk b/init/Android.mk
index 96b87316d..416b732d8 100644
--- a/init/Android.mk
+++ b/init/Android.mk
@@ -26,12 +26,10 @@ endif
 
 ifneq (,$(filter eng,$(TARGET_BUILD_VARIANT)))
 init_options += \
-    -DSHUTDOWN_ZERO_TIMEOUT=1 \
-    -DSPOOF_SAFETYNET=0
+    -DSHUTDOWN_ZERO_TIMEOUT=1
 else
 init_options += \
-    -DSHUTDOWN_ZERO_TIMEOUT=0 \
-    -DSPOOF_SAFETYNET=1
+    -DSHUTDOWN_ZERO_TIMEOUT=0
 endif
 
 init_options += -DLOG_UEVENTS=0 \
diff --git a/init/property_service.cpp b/init/property_service.cpp
index a84d22022..b2be381b3 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -1168,9 +1168,7 @@ void PropertyInit() {
     // checks pass. This needs to be done before parsing the kernel cmdline as
     // these properties are read-only and will be set to invalid values with
     // androidboot cmdline arguments.
-    if (SPOOF_SAFETYNET) {
-        SetSafetyNetProps();
-    }
+    SetSafetyNetProps();
 
     // If arguments are passed both on the command line and in DT,
     // properties set in DT always have priority over the command-line ones.
-- 
2.25.1

